### PR TITLE
story #3663 eCommerce: Portal is only visible to authenticated users

### PR DESCRIPTION
**Pull request by jrbirch99**
- Make the security group property on the "user types" field extendable so it can be overwritten in other modules

### CODE REVIEW

**Review by armandcela — APPROVED** [review]

**Review by adamrpatrick — APPROVED** [review]

File: odoo/addons/base/models/res_users.py [hunk 1]
```diff
@@ -1243,6 +1243,13 @@ def _update_user_groups_view(self):
             new_context.pop('install_filename', None)  # don't set arch_fs for this computed view
             new_context['lang'] = None
             view.with_context(new_context).write({'arch': xml_content})
+    
+    def _authorisation_for_user_type_fields(self):
+        """
+        Group that can view the user type. This has been put in a
+        method so it can overwritten.
```
Comment (adamrpatrick) [inline odoo/addons/base/models/res_users.py]: Super nit: "overridden" instead of "overwritten"?

**Comment by PeterAlabaster** [comment]
Q: Is this not possible to do with inheritance on `res.users`? Just anxious to make changes to core

**Comment by PeterAlabaster** [comment]
> Q: Is this not possible to do with inheritance on `res.users`? Just anxious to make changes to core

Sorry, just realised you're making it extensible here and not changing anything. Disregard that comment

**Review by PeterAlabaster — APPROVED** [review]

File: odoo/addons/base/models/res_users.py (hunk 1 above)
Comment (jrbirch99) [reply odoo/addons/base/models/res_users.py]: yep changed :slightly_smiling_face:

**Pull request merged**